app/models/comment.rb

```diff
@@ -207,6 +207,10 @@ def readable_publish_date
     end
   end
 
+  def hover_publish_date
+    created_at.strftime("%B %e %Y %I:%M %P %Z")
```

Comment:
How about naming this method `publish_date`?

To me, `hover_` couples this method to the way it's being presented to the user _(via a hovering action)_. `publish_date` would allow us to have a more generic name, which would still apply if at some point we'd like to display this same value in a different UI manner.

Moreover, we already have `readable_publish_date`. This longer version might make sense to consider it the full `publish_date`.

Thoughts?